### ISSUE TITLE
Trusted indicator

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/retrolab#276]
### Problem

The classic notebook shows a "Trust" indicator in the top area:

![image](https://user-images.githubusercontent.com/591645/141128068-f5bfecdd-5796-4533-889d-229bd9ed7b99.png)

RetroLab currently doesn't provide such indicator.

### Proposed Solution

JupyterLab doesn't have a visual indicator, but exposes a command to trust a notebook:

![image](https://user-images.githubusercontent.com/591645/141128303-8e801fd3-88ad-416e-950d-f20acd3a5a19.png)

We can reuse the same command, and add a new button to the `menu` area to the right of the kernel status indicator:

https://github.com/jupyterlab/retrolab/blob/9010ff543538dd6b15d647dd38ce44626095db84/packages/notebook-extension/src/index.ts#L165

### Additional context

This indicator can be added to the `notebook-extension`: https://github.com/jupyterlab/retrolab/blob/main/packages/notebook-extension/src/index.ts

**Comment by krassowski** [comment]
JupyterLab shows a shield on the statusbar (but there are some issues that we need to fix notebooks often being marked as not trusted even though those should be trusted.

**Comment by jtpio** [comment]
Ah thanks @krassowski :+1:

For some reason it has never been really visible to me. Probably because it's "down" in the UI and easy to miss.

I have updated the top comment to mention that.

**Comment by jweill-aws** [comment]
The [Jupyter Notebook security docs](https://jupyter-notebook.readthedocs.io/en/latest/security.html) suggest that JupyterLab also makes a menu option in the File menu available for trusting an untrusted notebook. Will there be a separate PR for the new menu option?

**Comment by krassowski** [comment]
Good point. JupyterLab has this option in the command palette but not in the menu.

While I don't see a mention of JupyterLab in that link, I see the point. I guess the problem with the docs in `jupyter-notebook` subdomain is that it sometimes speaks for Jupyter Notebook as a code/product and sometimes it makes broad statements which can or should be interpreted as "Jupyter as a whole", while not having a normative power itself. The current docs for `jupyter-server` at https://jupyter-server.readthedocs.io/en/latest/operators/security.html also contain the same mention.
